assets/css/frontend-form/layout2.css

```
@@ -29,48 +56,50 @@ body.mceContentBody{
-  color: #F20000;
+  color: var(--wpuf-l2-danger);
 }
 .wpuf-form-add.wpuf-form-layout2 ul.wpuf-form li .wpuf-fields input:not([type="file"]):not([type="radio"]):not([type="checkbox"]):not([type="submit"]),
 .wpuf-form-add.wpuf-form-layout2 ul.wpuf-form li .wpuf-fields textarea {
   width: 100% !important;
-  background: #292C3F;
+  background: var(--wpuf-l2-surface-bg);
   -moz-box-shadow: none;
   -webkit-box-shadow: none;
   box-shadow: none;
   border: 0;
```

Stylelint (17.14.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)